rcl/src/rcl/logging.c

```diff
@@ -64,10 +69,17 @@ rcl_logging_ext_lib_output_handler(
   const char * format, va_list * args);
 
 rcl_ret_t
-rcl_logging_configure(const rcl_arguments_t * global_args, const rcl_allocator_t * allocator)
+rcl_logging_init(const rcl_arguments_t * global_args, const rcl_allocator_t * allocator)
 {
+  pthread_mutex_lock(&g_rcl_logging_mutex);
+  if (g_rcl_logging_initialized) {
+    RCUTILS_LOG_WARN_NAMED("rcl.logging", "rcl_logging_init is already done.");
```

Comment:
You can't log if logging is not inited.
Use `fprintf` in this case.

Reply:
got it. i will use fprint.